package.json

```diff
@@ -28,7 +28,8 @@
   },
   "license": "MIT",
   "config": {},
-  "dependencies": {},
-  "devDependencies": {},
+  "devDependencies": {
+    "jquery": "^3.7.1"
+  },
```

Comment:
was `jquery` intentionally listed as a `devDependency` here? Currently the `package.json` is only used to allow for installation of the custom CSS. Can you explain your development workflow that requires the dependency to be added here?

Reply:
Yes, otherwise the IDE complains about undefined names, such as `ajax` if I recall.